lib/user.js

```diff
@@ -1,87 +1,15 @@
 'use strict';
 
-var _ = require('lodash');
-var shortid = require('shortid');
-var users = require('./data.json');
-
-module.exports = function(mediator) {
-  var topicList = 'wfm:user:list';
-  console.log('Subscribing to mediator topic:', topicList);
-  mediator.subscribe(topicList, function() {
-    setTimeout(function() {
-      mediator.publish('done:' + topicList, users);
-    }, 0);
-  });
-
-  var topicLoad = 'wfm:user:read';
-  console.log('Subscribing to mediator topic:', topicLoad);
-  mediator.subscribe(topicLoad, function(id) {
-    setTimeout(function() {
-      var user = _.find(users, function(_user) {
-        return _user.id === id;
-      });
-      if (user) {
-        mediator.publish('done:' + topicLoad + ':' + id, user);
-      } else {
-        mediator.publish('error:' + topicLoad + ':' + id, 'No such user');
-      }
-    }, 0);
-  });
-
-  var topicUsernameLoad = 'wfm:user:username:read';
-  console.log('Subscribing to mediator topic:', topicUsernameLoad);
-  mediator.subscribe(topicUsernameLoad, function(username) {
-    setTimeout(function() {
-      var user = _.find(users, function(_user) {
-        return _user.username === username;
-      });
-      if (user) {
-        mediator.publish('done:' + topicUsernameLoad + ':' + username, user);
-      } else {
-        mediator.publish('error:' + topicUsernameLoad + ':' + username, 'No such user');
-      }
-    }, 0);
-  });
-
-  var topicSave = 'wfm:user:update';
-  console.log('Subscribing to mediator topic:', topicSave);
-  mediator.subscribe(topicSave, function(user) {
-    setTimeout(function() {
-      var index = _.findIndex(users, function(_user) {
-        return _user.id === user.id;
-      });
-      users[index] = user;
-      console.log('Saved user:', user);
-      mediator.publish('done:' + topicSave + ':' + user.id, user);
-    }, 0);
-  });
-
-  var topicCreate = 'wfm:user:create';
-  console.log('Subscribing to mediator topic:', topicCreate);
-  mediator.subscribe(topicCreate, function(user) {
-    setTimeout(function() {
-      user.id = shortid.generate();
-      users.push(user);
-      console.log('Created user:', user);
-      mediator.publish('done:' + topicCreate + ':' + user.createdTs, user);
-    }, 0);
-  });
-
-  var topicDelete = 'wfm:user:delete';
```

Comment:
Those listeners are being added by the [`ArrayStore.prototype.listen` call](https://github.com/feedhenry-raincatcher/raincatcher-user/pull/19/files#diff-2c7d8efd78848f1f8e6d2eb8bc78d4c5R131).